CenterForOpenScience/ember-osf-web
# [ENG-3811] Handle fetching errors

-   Ticket: [ENG-3811]
-   Feature flag: n/a

## Purpose

Catch some errors when fetching resources

## Summary of Changes

1. Catch errors when loading a list of files from File and all of the provider-files.
2. Catch error loading the providers from the route
3. Report error when provider  is not passed to the manager (which usually means a problem loading the provider)
4. Add error messages to translations file

## Side Effects

Shouldn't be.

## QA Notes

We should see some error reporting in the sample project when trying to load dataverse and when trying to load a specific folder in Google Drive (`/Brian/Edge` I believe).


[ENG-3811]: https://openscience.atlassian.net/browse/ENG-3811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
